src/Stratis.Bitcoin.Api.Tests/ApiSettingsTest.cs

```diff
@@ -21,6 +22,11 @@ public ApiSettingsTest()
             Block.BlockSignature = false;
         }
 
+        private string GetTestDataDir([CallerMemberName] string path = null)
```

Comment:
@quantumagi TestBase has already got a directory helper method called `CreateTestDir`. There are a few other ones you can use as well. Have a look...

Reply:
The idea behind GetTestDataDir is to retain the current behavior. Using the other method leads to the following error on some of the test cases:
`Message: System.IO.IOException : The process cannot access the file '_DBreezeSchema' because it is being used by another process.`

```
        private string GetTestDataDir([CallerMemberName] string path = null)
        {
            string s1 = System.IO.Path.Combine("TestData", path);
            string s2 = CreateTestDir(this);
            return s2;
        }
```

Will leave the code as is for now. We can change this and make the required fixes in a follow-up PR.